rasa/shared/core/flows/flow.py

```diff
@@ -580,6 +580,33 @@ def as_json(self) -> Dict[Text, Any]:
         return dump
 
 
+@dataclass
+class BranchFlowStep(FlowStep):
+    """Represents the configuration of a branch flow step."""
+
```

Comment:
What's a branch flow step? Can't any flow step have branching next patterns?

Reply:
yes, but this one is special: it can ONLY have a branching pattern :) 

https://12749--rasahq-docs-rasa-v2.netlify.app/docs/rasa/flows/#branch